source/css/_common/scaffolding/base.styl

```diff
@@ -16,7 +16,6 @@ body {
   line-height: $line-height-base;
   min-height: 100%;
   position: relative;
-  transition: padding $transition-ease;
```

Comment:
This transition is necessary for sidebar open & close in scheme Muse & Mist. See `source/css/_schemes/Muse/sidebar.styl` (`padding-right` for `body.sidebar-active`)